app/views/transactions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<section class="min-h-screen bg-gray-200 flex flex-col items-center w-full">
+  <div class="bg-main w-full text-white pt-2">
+    <%= link_to category_transactions_path(@category), class: "text-white ps-3" do %>
+      <i class="fa-solid fa-arrow-left fa-lg"></i>
+    <% end %>
+    <div class="h-10 bg-main w-full text-white flex justify-center items-end pb-2">
+      <h1 class="font-bold">Add a New Transaction</h1>
```

Comment:
- [ ] I understand that here when you add a new transaction, it places itself in the current category you are adding it to. But, according to the project requirements, the form should include an option to select the category
![image](https://github.com/SamTush/budget/assets/45460192/f706e3cc-7096-45af-997e-8fb184c809d4)